From 2c1f01d2496c90a5f4a650dff7d59fe09a5cb09d Mon Sep 17 00:00:00 2001
From: jenisys <jenisys@users.noreply.github.com>
Date: Sun, 12 Sep 2021 16:07:32 +0200
Subject: [PATCH] FIX #955: setup: Remove attribute 'use_2to3'

REASON:
* This attribute is deprecated since setuptools >= v58.0.2 (2021-09-06).
* 2to3 conversion should not be needed anymore.
  Currently, code should run on python2 and python3 (by using six, etc.).
---
 setup.py | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/setup.py b/setup.py
index fd89bda..ba407fd 100644
--- a/setup.py
+++ b/setup.py
@@ -118,8 +118,8 @@ setup(
             "pylint",
         ],
     },
-    # MAYBE-DISABLE: use_2to3
-    use_2to3= bool(python_version >= 3.0),
+    # DISABLED: use_2to3= bool(python_version >= 3.0),
+    # DEPRECATED SINCE: setuptools v58.0.2 (2021-09-06)
     license="BSD",
     classifiers=[
         "Development Status :: 4 - Beta",
@@ -129,12 +129,11 @@ setup(
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.3",
-        "Programming Language :: Python :: 3.4",
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: Implementation :: CPython",
         "Programming Language :: Python :: Implementation :: Jython",
         "Programming Language :: Python :: Implementation :: PyPy",
